src/aiai_eval/task_configs.py

```diff
@@ -165,8 +165,8 @@ def get_all_dataset_tasks() -> Dict[str, DatasetTask]:
 SENT = DatasetTask(
     name="sent",
     dataset_name="angry-tweets",
-    pretty_dataset_name="the truncated version of AngryTweets",
-    huggingface_id="ScandEval/angry-tweets-mini",
+    pretty_dataset_name="Angry Tweets, Danish Twitter data",
```

Comment:
I think having descriptions like that would look a bit weird in the logging. We are going to add proper descriptions to the website in any case 🙂 

```suggestion
    pretty_dataset_name="Angry Tweets",
```